ihatemoney/web.py

```diff
@@ -328,17 +338,25 @@ def remind_password():
             # get the project
             project = Project.query.get(form.id.data)
             # send a link to reset the password
-            current_app.mail.send(
-                Message(
-                    "password recovery",
-                    body=render_localized_template(
-                        "password_reminder", project=project
+            remind_message = Message(
+                "password recovery",
+                body=render_localized_template("password_reminder", project=project),
+                recipients=[project.contact_email],
+            )
+            success = send_email(remind_message)
+            if success:
+                return redirect(url_for(".password_reminder_sent"))
+            else:
+                flash(
+                    _(
+                        "Sorry, there was an error while sending you an email "
+                        "with password reset instructions. "
+                        "Please check the email configuration of the server "
+                        "or contact the administrator."
```

Comment:
What do you think of writing the email address from `MAIL_DEFAULT_SENDER`?
It *should* be reliable, but not always. However, I am always puzzled by message that says to contact the administrator without a clue on who it can be.
Maybe we can add:
> contact the administrator. Try send an email to `MAIL_DEFAULT_SENDER[1]`.

Reply:
Hmm, good point. For I hate money I expect users would know their administrators (small instances), but this may not always be the case.

I'm a bit hesitant because it reveals some (potentially private) information. True, this information is used as `From:` for outgoing emails, but still.

Maybe only do that if public project creation is allowed? Also, we should avoid showing the email address if it's budget@notmyidea.org ;)

In any case, I suggest doing that in another PR. The initial goal of this one was to fix a crash, and it is the last blocking point before going ahead with the stable release (#588)

Reply:
See #626